pelorus-operator/bundle/manifests/charts.pelorus.konveyor.io_pelorus.yaml

```diff
@@ -15,24 +15,56 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: Pelorus is the Schema for the pelorus API
+        description: Pelorus is the Schema for pelorus API to adopt instance to the
```

Comment:
Grafana operator's description are like this, but would not be better to have something like "Configure a running instance of Pelorus" (like Prometheus operator)?